### PR TITLE
add avatar, alert and badge component

### PR DESCRIPTION
**Pull request by Kishanhs1990**
Component Library
1. Avatar component
2. Alert component
3. Badge component

### CODE REVIEW

File: components/alert.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      </div>
+      <select>
+        <option value="0x" selected>Select severities:</option>
+        <option value="1x">Primary</option>
+        <option value="2x">Error</option>
+        <option value="3x">Success</option>
+        <option value="4x">Warning</option>
+      </select>
+      <script>
+        $("select").on("change", function () {
```
Comment (deekshithmd) [inline components/alert.html]: keep js code in seperate file

File: css/alert.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  min-height: 16px;
+  width: 30%;
+  margin: 16px 0;
+  padding: 16px 24px;
+  line-height: 22px;
+}
+
+.primary-alert {
+  background-color: #dff0ff;
+  color: #2185d0;
```
Comment (deekshithmd) [inline css/alert.css]: try to use color variables

File: css/main.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+.margin-left {
+  margin-left: 28px;
+}
+
+.font-color {
+  color: var(--cta-color);
+}
+
+.banner-styled {
+  font-size: 80px;
```
Comment (deekshithmd) [inline css/main.css:34]: using rem will be good

**Review by deekshithmd — APPROVED** [review]

File: components/alert.html (hunk 1 above)
Comment (Kishanhs1990) [reply components/alert.html]: has been updated as per comment

File: css/alert.css (hunk 2 above)
Comment (Kishanhs1990) [reply css/alert.css]: has been updated as per the comment

File: css/main.css (hunk 3 above)
Comment (Kishanhs1990) [reply css/main.css:34]: will update rem in second pr for component library

File: css/badge.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+.icon-container {
+  margin: 0 300px;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: flex-end;
+  margin-bottom: 20px;
+}
+
+.primary {
+  background-color: #dff0ff;
```
Comment (kushalsourav) [inline css/badge.css:25]: use var for colors

**Review by kushalsourav — APPROVED** [review]

File: css/badge.css (hunk 4 above)
Comment (Kishanhs1990) [reply css/badge.css:25]: updated thanks

**Pull request merged**